Dotnet.Homeworks.Tests/Docker/DockerTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Dotnet.Homeworks.Tests.RunLogic.Attributes;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Dotnet.Homeworks.Tests.Docker;
+
+public class DockerTests
+{
+    private const string FileName = "docker-compose.yml";
+
+    private static readonly Lazy<DockerCompose> DockerComposeDeserializedFactory = new(() =>
+    {
+        var filePath = Path.Combine(TryGetSolutionDirectoryInfo().FullName, FileName);
+
+        if (!File.Exists(filePath)) throw new InvalidOperationException();
+
+        var b = new DeserializerBuilder()
+            .IgnoreUnmatchedProperties()
+            .WithNamingConvention(UnderscoredNamingConvention.Instance);
+
+        return b.Build().Deserialize<DockerCompose>(File.ReadAllText(filePath));
+    });
+
+    private static readonly DockerCompose DockerComposeDeserialized = DockerComposeDeserializedFactory.Value;
+
+    [Homework(RunLogic.Homeworks.Docker)]
+    public void ConnectionStringsDefault_IsNotNullInDockerCompose()
+    {
+        var dotnetWebConnectionString =
+            DockerComposeDeserialized.Services?.DotnetWeb?.Environment?.GetValueOrDefault("ConnectionStrings__Default");
```

Comment:
Тут точно не должна быть 1 черта?
```suggestion
            DockerComposeDeserialized.Services?.DotnetWeb?.Environment?.GetValueOrDefault("ConnectionStrings_Default");
```

Reply:
Точно не должно быть __